test/config/hashrings.json

```diff
@@ -5,7 +5,7 @@
             "1610b0c3-c509-4592-a256-a1871353dbfa"
         ],
         "endpoints": [
```

Comment:
Can this file be delete if we are defining & running all e2e tests from Go?

Reply:
Static config files (like this one or the Loki config) from the `config` directory are still used. Only dynamic configs (e.g. requiring container hostname) are being generated during the test, static ones are copied into the shared directory.